main.cpp

```diff
@@ -50,108 +50,107 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
-
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+ // Noun: fish
+//  action 1: The fish eats
+fish.eat();
+//  action 2: The fish swims 
+fish.swim();
+//  action 3: The fish breeds
+fish.breed();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: dog
+//  action 1: the dog barks
+dog.bark();
+//  action 2: the dog plays
+dog.play();
+//  action 3: the dog sit
+dog.sit();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: plane
+//  action 1: the ship departs
+ship.depart();
+//  action 2: the ship sails
+ship.sail();
+//  action 3: the ship docks
+ship.dock(); 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: car wash
+//  action 1: the carwash shampoos and waxes
+carWash.shampooAndWax();
+//  action 2: the carwash rinses
+carWash.rinse();
+//  action 3: the car wash blow dries
+carWash.blowDry();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: printer
+//  action 1: the printer prints
+printer.print();
+//  action 2: the printer scans
+printer.scan();
+//  action 3: the printer cleans itself
+printer.clean();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: engine
+//  action 1: the engine starts
+engine.start();
+//  action 2: the engine accelerates
+engine.accelerate();
+//  action 3: the engine stops
+engine.stop();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: tape recorder
+//  action 1: the tape recorder records
+tapeRecorder.record();
+//  action 2: the tape recorder rewinds
+tapeRecorder.rewind();
+//  action 3: the tape recorder plays
+tapeRecorder.play();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: coffee machine
+//  action 1: the coffee machine grinds beans
+coffeeMachine.grindBeans();
+//  action 2: the coffee machine heats water 
+coffeeMachine.heatWater();
+//  action 3: the coffee machine brews coffee
+coffeeMachine.brewCoffee();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: camera
+//  action 1: the camera focuses
+camera.focus();
+//  action 2: the camera zooms 
+camera.zoom();
+//  action 3: the camera captures and image
+camera.captureImage();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.